frontend/src/components/QuestionPanel/styles.scss

```
@@ -0,0 +1,64 @@
+@import '../../styles/functions.scss';
+
+.question-panel {
+    --item-question-width: 198px;

```

У нас, в інших файлах стилів, між властивостями й  змінними порожня лінія 
```suggestion
    --item-question-width: 198px;

```